framework/build.gradle

```diff
@@ -29,6 +34,13 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.13'
     testImplementation project(':framework-test')
     testImplementation sourceSets.testannotations.output
+
+    // AutoValue support in Returns Receiver Checker
+    testImplementation "com.google.auto.value:auto-value-annotations:${versions.autoValue}"
+    testImplementation "com.google.auto.value:auto-value:${versions.autoValue}"
+
+    // Lombok support in Returns Receiver Checker
+    testImplementation "org.projectlombok:lombok:${versions.lombok}"
```

Comment:
> Overall this is in good shape. I had a few minor comments throughout.
> 
> My big concern is the testing for lombok. Right now, the code being checked in has already been delombok'd, but lombok is listed as a dependency in the build file. We should either:
> 
> 1. keep the delombok'd test code, and remove lombok from the build file, or
> 2. create/find the pre-lombok source files that correspond to the delombok'd tests here, and then write a script that delomboks them and make that script a prerequisite for running these tests
> 
> I prefer option 2, because it will catch any breaking changes in new lombok versions. I'm willing to help to do this. Option 1 is easier, and removes a dependency, but it won't alert us if a new lombok version breaks RR support.

Regarding option 1, if we remove the Lombok from build file, we will face the error "lombok package is missing" since we are using the annotations in the tests. Would you please help me how can I remove it without facing this error, or do you prefer to keep it in the build file as it is.

Reply:
That's a good point that I had missed - Lombok does not distribute separate annotation and processor jars, like AutoValue. So if we keep things as-is, you're right that we'd need to keep the dependency. A note about why it's needed even though the annotation processor isn't being run would be helpful.

Reply:
Thank you for your help @kelloggm